index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width,initial-scale=1" />
+        <title>Portfolio</title>
+        <link href="style.css" rel="stylesheet" />
+        <script src="https://kit.fontawesome.com/0007bfca3c.js" crossorigin="anonymous"></script>
+    </head>
+    <body>
+        <header>
+            <nav class="navbar">
+                <a class="profile" href="#">Juan Gonzalez</a>
+                <a href="#"><i class="fas fa-bars"></i></a>
+            </nav>
+        </header>
+        <section class="introduction">
+            <div>
+                <h1 class="presentation">Hey There. I'm Juan</h1>
+                <h2 class="profesion">I'm a software developer</h2>
+                <p class="context">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.</p>
+            </div>
+            <ul class="social">
+                <li>
+                    <a href="https://github.com/JbirdL86" rel="noopener" target="_blank">
+                    <img src="assets/github.png" alt="github-icon" />
+                </a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/in/juan-luis-0551921aa/" rel="noopener" target="_blank">
+                        <img src="assets/linkd.png" alt="linkd-icon" />
+                    </a>
+                </li>
+                <li>
+                    <a href="https://hackernoon.com/u/JbirdL86" rel="noopener" target="_blank">
+                        <img src="assets/ig.png" alt="hackernoon-icon" />
+                    </a>
+                </li>
+                <li>
+                    <a href="https://twitter.com/JuanLui06498455" rel="noopener" target="_blank">
+                    <img src="assets/github.png" alt="twitter-icon" />
+                    </a>
+                </li>
+                <li>
+                    <a href="https://medium.com/@jgonzalez.wj" rel="noopener" target="_blank">
+                        <img src="assets/medium.png" alt="medium-icon" />
+                    </a>
+                </li>
+            </ul>
+        </section>
```

Comment:
- [ ] As I've mentioned above, kindly use the font type as defined in the design.